packager/Dockerfile

```diff
@@ -1,23 +1,56 @@
-# Maybe switch to one of the Google-provided images? But seems to be go 1.8, not 1.10:
-# https://cloud.google.com/appengine/docs/flexible/custom-runtimes/build#base
-FROM golang:1.15
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
-ADD https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_amd64 /usr/local/bin/dumb-init
-RUN chmod +x /usr/local/bin/dumb-init
-ENTRYPOINT ["dumb-init", "--"]
+# Use an official Go runtime as a parent image
+FROM golang:1.13
 
-RUN groupadd -r amppkg && useradd --no-log-init -r -g amppkg amppkg
-WORKDIR /amppkg
+ENV GO111MODULE=on
 
-COPY amppkg.toml /amppkg
-# Copy whatever certs are available in the certs/ directory  into the container. 
-COPY certs/*pem* /amppkg/
+# Install AMP Packager
+# Run this if you want to go off master branch.
+# RUN go get -v github.com/ampproject/amppackager/cmd/amppkg@master
+RUN go get -v github.com/ampproject/amppackager/cmd/amppkg
+
+# Install git
+RUN apt-get update \
+    && apt-get install -y git
+
+# Clone amppackager files so we can use b3 directory
```

Comment:
NIT: "b3 directory" is ambiguous, and redundant as it is immiedtly pointed out 5 lines later. It would be great to say _what_ the b3 directory is for